src/coreclr/jit/gentree.cpp

```diff
@@ -26877,6 +26980,10 @@ bool GenTreeHWIntrinsic::OperIsEmbRoundingEnabled() const
         case NI_AVX512F_FusedMultiplySubtractNegated:
         case NI_AVX512F_FusedMultiplySubtractNegatedScalar:
         case NI_AVX512F_FusedMultiplySubtractScalar:
+        case NI_AVX10v1_FusedMultiplyAddScalar:
+        case NI_AVX10v1_FusedMultiplyAddNegatedScalar:
+        case NI_AVX10v1_FusedMultiplySubtractScalar:
+        case NI_AVX10v1_FusedMultiplySubtractNegatedScalar:
```

Comment:
Are we missing `NI_AVX10v1_FusedMultiplyAdd` and friends too?

`Avx10v1` implies `AVX2 + FMA` but can only inherit from FMA and so this seems to fall into the category we said we'd duplicate (much like `Avx512DQ` being duplicated in `Avx10v1.V512`)

Reply:
I think you are right. Currently only the `*scalar` versions(for eg **FusedMultiplyAddNegatedScalar**()) have been added to `Avx10v1`. I think everything in [here](https://github.com/dotnet/runtime/blob/362a95d53d7266ee9b7452e351d07845dcc1edd2/src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Fma.cs) should be added. But assume the naming should be consistent with [this](https://github.com/dotnet/runtime/blob/362a95d53d7266ee9b7452e351d07845dcc1edd2/src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Avx512F.cs#L2499-L2659)

Reply:
I have made the required changes. Thanks for the calrification @DeepakRajendrakumaran